data/json/mutations/mutations.json

```diff
@@ -1203,7 +1233,6 @@
     "points": -2,
     "description": "Without reading glasses, you are unable to read anything, and take penalties on melee accuracy and electronics/tailoring crafting.  However, you are guaranteed to start with a pair of reading glasses.",
     "starting_trait": true,
-    "category": [ "BATRACHIAN" ],
```

Comment:
I think I meant to have frogs be near sighted.  Thanks for catching this.

Reply:
I was a tad confused why they were both, so I looked it up and it seemed so, yeah.